Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -225,7 +225,7 @@ def get_list_response(path, method='get', limit=None, body={}, params={}):
 def get_last_scan(asset):
     if asset['history'] is None:
         return "-"
-    sorted_dates = sorted(asset['history'], key=lambda h: time.strptime(h['date'], "%Y-%m-%dT%H:%M:%S.%fZ"),
+    sorted_dates = sorted(asset['history'], key=get_datetime_from_asset_history_item,
```

Comment:
will the sorting work correctly if some of the strings are with milliseconds and some aren't?
please test this.

Reply:
I'll add a test for it, however it shouldn't affect the sort, as datetime is sorted by its timestamp